src/Controller/LoginController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\UserManager;
+
+class LoginController extends AbstractController
+{
+    public function login(): string
+    {
+        $errors = [];
+        $connexion = [];
+
+        if ($_SERVER["REQUEST_METHOD"] === "POST") {
+            $connexion = array_map('trim', $_POST);
+            $errors = $this->validate($connexion);
+        }
+        return $this->twig->render('Login/login.html.twig', [
+            'errors' => $errors,
+        ]);
+    }
+
+    public function logout()
+    {
+        if (!empty($_SESSION['user'])) {
+            unset($_SESSION['user']);
+        }
+
+        header('Location: /');
+    }
+
+    private function validate(array $connexion): array
+    {
+        $errors = [];
+        if (empty($connexion['email'])) {
+            $errors[] = 'Le champ email ne doit pas être vide';
+        }
+        if (empty($connexion['password'])) {
+            $errors[] = 'Le mot de passe ne doit pas être vide';
+        }
+        if (!filter_var($connexion['email'], FILTER_VALIDATE_EMAIL)) {
+            $errors[] = "format d'email invalide";
+        }
+        if (empty($errors)) {
+            $userManager = new UserManager();
+            $user = $userManager->selectOneByEmail($connexion['email']);
```

Comment:
![look-you-see](https://user-images.githubusercontent.com/79161810/167582911-0a538eab-149a-4fa9-b88c-52858ac4c46b.gif)